components/brave_wallet/browser/blockchain_registry.cc

```diff
@@ -298,4 +304,8 @@ void BlockchainRegistry::GetCoingeckoId(const std::string& chain_id,
   std::move(callback).Run(GetCoingeckoId(chain_id, contract_address));
 }
 
+bool BlockchainRegistry::IsOfacAddress(const std::string& address) {
+  return ofac_addresses_.find(address) != ofac_addresses_.end();
```

Comment:
base::Contains

Reply:
Also ofac_addresses_ should be filled with lowercase addresses and IsOfacAddress should search for lowercased arg

Reply:
Good call